website/content/getting-started.mdx

```diff
@@ -0,0 +1,344 @@
+---
+title: 'Getting Started'
+---
+
+# Getting Started
+
+**Start here if you're new to Armada. Learn how to install it and run your first job.**
+
+## 2.1 Quickstart
```

Comment:
This Quickstart is outdated/not working.

Official quickstart is via the Armada Operator - https://github.com/armadaproject/armada-operator?tab=readme-ov-file#quickstart

Reply:
Refactored